src/controllers/account.ts

```diff
@@ -42,3 +43,35 @@ export async function deleteYourProofOfIdGet(
 ): Promise<void> {
   res.render("account/delete-proof-of-identity");
 }
+
+export async function deleteYourProofOfIdPost(
+  req: Request,
+  res: Response
+): Promise<void> {
+  const session = await getSessionFromStorage(req.session?.sessionId);
+  if (session) {
+    const kvbUri = await getDatasetUri(
+      session,
+      "private/govuk/identity/poc/credentials-pat/vcs/kbv/metadata"
```

Comment:
We're using this path in a few places now, we should consider moving it to a helper function somewhere (probably `src/lib/pod.ts`. You don't need to do that now though

Reply:
yeah good shout!